install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -175,6 +175,14 @@ data:
         - name: ISTIO_META_NETWORK
           value: "{{ .Values.global.network }}"
         {{- end }}
+        - name: ISTIO_META_WORKLOAD_SDS_ENABLED
+          value: "{{ .Values.global.sds.enabled }}"
+        - name: ISTIO_META_K8S_JWT_PATH
```

Comment:
Any option to not have an token ( if users have SPIRE for example, which uses UDS and docker instead of token ) ?

Reply:
(can be done later, just curious )

Reply:
could provide such support in the future, right now we depend on token.